CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.0-nullsafety
+   * Upgrade to sound null safety
+   * Require Dart 2.12.0-0
+
 ## 1.0.4-dev
```

Comment:
Delete this entry, then. We never published 1.0.4

Reply:
Done